src/components/List/NormalList/NormalList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import ListItem from '../ListItem';
+import { generateUniqueID } from 'utils/helpers';
+import { listStyle } from './NormalList.style';
+import { SelectOption } from '../../Select/Select';
+import { isSelected } from '../utils';
+
+type Props = {
+  items: (string | number | SelectOption)[];
+  /** Size of the list's row (height of ListItem Component)  */
+  rowSize: 'small' | 'normal';
+  /** Width of the list */
+  width?: number;
+  /** Height of the list */
+  height?: number;
+  /** Ref of ListItem component */
+  listItemRef?: React.RefObject<HTMLDivElement>;
+  /** Selected Item */
+  selectedItem?: string | number;
+  /** Search Term to be highlighted in list items */
+  searchTerm?: string;
+  /** Option Click handler for SelectOption[] data case */
+  handleOptionClick?: (option: SelectOption) => void;
+  /** Data Test Id Prefix */
+  dataTestIdPrefix?: string;
```

Comment:
same here for dataTestId - sorry for duplication of comments need them to track this